pkg/render/kubecontrollers/kube-controllers.go

```diff
@@ -332,7 +332,7 @@ func kubeControllersRoleEnterpriseCommonRules(cfg *KubeControllersConfiguration)
 			// Needed to validate the license
 			APIGroups: []string{"crd.projectcalico.org"},
 			Resources: []string{"licensekeys"},
-			Verbs:     []string{"get"},
+			Verbs:     []string{"get", "watch", "list"},
```

Comment:
Watch is sufficient.

Reply:
@robbrockbank what are you thoughts on keeping the list (so that it matches the level of permissions of the `projectcalico.org` version of `licensekeys ` right above this?

I suppose there might be a need for list in the managed clusters case.

Reply:
Yeah, nevermind. The managed clusters version definitely needs the list.